src/util/numerics.test.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as N from "./numerics";
+
+describe("util/numerics", () => {
+  describe("finite", () => {
+    it("allows finite numbers", () => {
+      for (const f of [-1.2, 0, 999]) {
```

Comment:
Would be good to test out some irrational numbers too like Math.PI and repeating decimals like `5/7`.